esmvalcore/config/config_defaults/config-user.yml

```diff
@@ -13,14 +13,6 @@
 # file.
 #
 ###############################################################################
-#
```

Comment:
What is the plan for the ESMValTool copy, remove?

Reply:
Yes, I think that's the easiest solution. We don't want users to simply copy-paste the file anymore but rather use `esmvaltool config ...`, so it doesn't need be located in the repo anymore.